src/main/flight/imu.c

```diff
@@ -106,7 +106,8 @@ PG_RESET_TEMPLATE(imuConfig_t, imuConfig,
     .dcm_ki = 0,                   // 0.003 * 10000
     .small_angle = 25,
     .accDeadband = {.xy = 40, .z= 40},
-    .acc_unarmedcal = 1
+    .acc_unarmedcal = 1,
+    .use_gps_heading = 0,
```

Comment:
Using `use_gps_heading` will make the code clearer. (`.use_gps_heading = false`).